### PR TITLE
Update head element section in html-boilerplate.md

### PR DESCRIPTION
**Pull request by yogeshwarbari**
There is a statement "Anything included within the head element will not be displayed to the user." which may be misinterpreted by beginners that "whatever you write inside <head> and </head> tags even the plain text should not be displayed on webpage by browser." which is wrong. I think the statement might create this misconception which will remain in their mind until they try stuff like putting random text in head tag and then they can see the output on screen.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

...your text here

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: foundations/html_css/html-foundations/html-boilerplate.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Back in the `index.html` lets add the `<html>` element by typing out its opening
 
 ### Head Element
 
-The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser. Anything included within the head element will **not** be displayed to the user.
+The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser.
```
Comment (fortypercenttitanium) [inline foundations/html_css/html-foundations/html-boilerplate.md:52]: I like the idea behind this change, but I don't like simply removing this sentence. Do you feel like you could reword it to be more accurate, while also emphasizing that elements within the `head` element *should not* be displayed to the user?

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]

File: foundations/html_css/html-foundations/html-boilerplate.md (hunk 1 above)
Comment (yogeshwarbari) [reply foundations/html_css/html-foundations/html-boilerplate.md:52]: When I raised the PR I was not able to think proper replacement for the removed statement but now after thinking some more time I can reword the correct statement.

**Review by fortypercenttitanium — APPROVED** [review]
Thank you!
